_src/css/main.css

```diff
@@ -0,0 +1,8248 @@
+@charset "UTF-8";
```

Comment:
Looks like you may have added this file?  perhaps it was created when building the app?

Reply:
Remove this file